espresso/devnet-tests/key_rotation_test.go

```diff
@@ -71,15 +80,42 @@ func TestChangeBatchInboxOwner(t *testing.T) {
 	// Change the BatchAuthenticator's owner
```

Comment:
Nit: Should we remove this comment since we don't transfer the ownership until line 107 now?